data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -325,6 +325,74 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [/message]
     [/event]
 
+    # Triggers the first time that the player has two healers
+    [event]
+        name=recruit
+        [filter]
+            side=1
+            type=Elvish Shaman
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                ability_type=heals
+                count=2
+            [/have_unit]
+        [/filter_condition]
+
+        [store_unit]
+            [filter]
+                ability_type=heals
+                [not]
+                    id=$unit.id
+                [/not]
+                {EVERYWHERE}
+            [/filter]
+            variable=other_healer
+        [/store_unit]
+
+        [if]
+            [variable]
+                name=other_healer.type
+                equals=Elvish Shaman
+            [/variable]
+            [then]
+                [message]
+                    speaker=$unit.id
+                    # po: both speaker and $other_healer are elvish shamans.
+                    # If there are allied sides then this isn't accurate, but at this stage of the tutorial allied sides haven't been introduced yet.
+                    message= _ "Both Shaman $other_healer.name and I can heal other units. However, each unit can only receive healing from one healer — even if we’re both next to a unit then that unit will still only heal 4 hitpoints per turn."
+                [/message]
+                [message]
+                    speaker=$unit.id
+                    # This line could be said by the other healer, but there's only one shaman portrait available, and it would be confusing to have dialog between two units with identical portraits.
```

Comment:
You might be able to use `[message]caption=`, or put $other_unit's portrait on the right, or both. Or cheat and use the Elvish Lady portrait.